--- jekyll-assets-3.0.12\lib\jekyll\assets\utils.rb.orig	Thu Mar 28 19:28:14 2019
+++ jekyll-assets-3.0.12\lib\jekyll\assets\utils.rb	Thu Mar 28 20:57:38 2019
@@ -250,7 +250,7 @@
 
         path = []
         path << cdn  if Jekyll.production? && cdn
-        path << base if Jekyll.dev? || !cdn || (cdn && cfg[:cdn][:baseurl])
+        #path << base if Jekyll.dev? || !cdn || (cdn && cfg[:cdn][:baseurl])
         path << dest if Jekyll.dev? || !cdn || (cdn && cfg[:cdn][:destination])
         path << user_path unless user_path.nil? || user_path == ""
 
